libs/dh/admin/feature-user-management/src/lib/drawer/roles/dh-role-drawer.component.html

```diff
@@ -24,14 +24,30 @@
 
     <watt-drawer-content *ngIf="drawer.isOpen">
       <div class="role-name__grid">
-        <h2 class="role-name__headline">{{ selectedRole?.name }}</h2>
+        <h2 class="role-name__headline">{{ basicUserRole?.name }}</h2>
         <dh-role-status
-          *ngIf="selectedRole"
-          [status]="selectedRole.status"
+          *ngIf="basicUserRole"
+          [status]="basicUserRole.status"
         ></dh-role-status>
       </div>
+      <ng-container *rxLet="isLoading$ as isLoading">
+        <ng-container *rxLet="hasGeneralError$ as hasGeneralError">
+          <dh-drawer-role-tabs
+            *ngIf="!isLoading && !hasGeneralError"
+            [role]="this.userRoleWithPermissions$ | push"
```

Comment:
nitpick: delete `this`

```suggestion
            [role]="userRoleWithPermissions$ | push"
```